src/libsyntax/parse/parser/expr.rs

```diff
@@ -844,29 +843,29 @@ impl<'a> Parser<'a> {
                 return self.parse_block_expr(None, lo, BlockCheckMode::Default, attrs);
             }
             token::BinOp(token::Or) | token::OrOr => {
-                return self.parse_lambda_expr(attrs);
+                return self.parse_closure(attrs);
```

Comment:
Seems this wasn't changed to `.parse_closure_expr`?

Reply:
Sorry, my bad. Just forgot.